<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheoryWindow</class>
 <widget class="QDialog" name="TheoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Теория | Ray Tracing | Pre-alpha 0.5v</string>
  </property>
  <widget class="QTextBrowser" name="text_theory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>741</width>
     <height>621</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeWidget" name="list_of_contents">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>20</y>
     <width>291</width>
     <height>481</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Содержание</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>Оглавление</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Геометрическая оптика</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Рендеринг</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Виды рендеринга</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Трассировка лучей</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пример</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="button_next_page">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>530</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Следующая страница</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/right-arrow.png</normaloff>images/right-arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_prev_page">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>530</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Предыдущая страница</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/left-arrow.png</normaloff>images/left-arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_help">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>600</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Получить помощь</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/help-button.png</normaloff>images/help-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
